samples/projection-management/Program.cs

```diff
@@ -0,0 +1,427 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using EventStore.Client;
+
+namespace projection_management {
+	public static class Program {
+
+		/// <summary>
+		/// This sample demonstrate the projection management features.
+		/// It will create data in the target database.
+		/// It will create a series of projections with the following content
+		/// fromAll() .when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();
+		/// </summary>
+		private static async Task Main() {
+
+			const string connection = "esdb://localhost:2113?tls=false";
+			var managementClient = ManagementClient(connection);
+
+			Console.WriteLine("Populate data");
+			await Populate(connection, 100);
+			Console.WriteLine("RestartSubSystem");
+			await RestartSubSystem(managementClient);
+			await Task.Delay(500); // give time to the subsystem to restart
+
+			Console.WriteLine("Disable");
+			await Disable(managementClient);
+			Console.WriteLine("Disable Not Found");
+			await DisableNotFound(managementClient);
+
+			Console.WriteLine("Enable");
+			await Enable(managementClient);
+			Console.WriteLine("Enable Not Found");
+			await EnableNotFound(managementClient);
+
+			Console.WriteLine("Delete");
+			await Delete(managementClient);
+
+			Console.WriteLine("Abort");
+			await Abort(managementClient);
+			Console.WriteLine("Abort Not Found");
+			await Abort_NotFound(managementClient);
+
+			Console.WriteLine("Reset");
+			await Reset(managementClient);
+			Console.WriteLine("Reset Not Found");
+			await Reset_NotFound(managementClient);
+
+			Console.WriteLine("CreateContinuous");
+			await CreateContinuous(managementClient);
+			Console.WriteLine("CreateContinuous conflict");
+			await CreateContinuous_Conflict(managementClient);
+			//Console.WriteLine("CreateOneTime");
+			//await CreateOneTime(managementClient);
+			Console.WriteLine("Update");
+			await Update(managementClient);
+			Console.WriteLine("Update_NotFound");
+			await Update_NotFound(managementClient);
+			Console.WriteLine("ListAll");
+			await ListAll(managementClient);
+			Console.WriteLine("ListContinuous");
+			await ListContinuous(managementClient);
+			Console.WriteLine("GetStatus");
+			await GetStatus(managementClient);
+			// Console.WriteLine("GetState");
+			// await GetState(managementClient);
+			Console.WriteLine("GetResult");
+			await GetResult(managementClient);
+		}
+
+
+
+		private static EventStoreProjectionManagementClient ManagementClient(string connection) {
+
+			#region createClient
+
```

Comment:
**FIX:** I think that those empty lines after #region create the redundant empty lines in docs, e.g.:
![obraz](https://user-images.githubusercontent.com/5562528/129678301-3631c902-d0d3-48b1-99fa-a8c98aa2e635.png)
![obraz](https://user-images.githubusercontent.com/5562528/129678376-5a83e794-57c6-4071-8661-2f06372c4340.png)